wordpress-mobile/WordPress-Android
# Issue/6438 login prologue analytics

### Fix
Add analytics to login prologue to track when users page through the screens as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6438.

### Test
1. Set breakpoints on each case of [login prologue paging](https://github.com/wordpress-mobile/WordPress-Android/blob/cf4f570749c1d11a71c47bcda86715253e7b4dc9/WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginPrologueFragment.java#L53-L67).
2. Swipe through login prologue screens.
3. Notice breakpoints are hit.
4. Notice [analytics events](https://github.com/wordpress-mobile/WordPress-Android/blob/cf4f570749c1d11a71c47bcda86715253e7b4dc9/libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java#L691-L700) are tracked.
